osbrain/tests/test_agent_sync_publications.py

```diff
@@ -283,3 +283,41 @@ def test_wait(nsproxy):
     assert wait_agent_attr(client, name='error_log', length=1, timeout=0.5)
     assert server.get_attr('received') == [fast, slow]
     assert 'x' + str(slow) not in client.get_attr('received')
+
+
+def test_various_requests(nsproxy):
```

Comment:
You are right, what I'm testing is way too abroad/doesn't add anything really. I'll close this PR. :+1: